sport/app/cricket/feed/CricketThrottler.scala

```diff
@@ -20,10 +19,18 @@ class CricketThrottlerActor()(implicit materializer: Materializer) extends Actor
 
   private case class TaskWithSender[+T](sender: ActorRef, task: () => Future[T])
 
+  val completionMatcher: PartialFunction[Any, CompletionStrategy] = { case e => CompletionStrategy.immediately }
+  val failureMatcher: PartialFunction[Any, Throwable] = { case e => new Throwable() }
+
   val throttler: ActorRef = Source
-    .actorRef[CricketThrottledTask[Nothing]](bufferSize = 1024, OverflowStrategy.dropNew)
+    .actorRef[CricketThrottledTask[Nothing]](
+      completionMatcher = completionMatcher,
+      failureMatcher = failureMatcher,
```

Comment:
Forgive my laziness for not googling, what do these do?

Reply:
I am assuming you were asking about the agent itself. It retrieves cricket  data from PA :)

Reply:
Otherwise we are setting run parameters for the agent
```
  val completionMatcher: PartialFunction[Any, CompletionStrategy] = { case e => CompletionStrategy.immediately }
  val failureMatcher: PartialFunction[Any, Throwable] = { case e => new Throwable() }
```